aten/src/ATen/cpu/vec256/vec256_int.h

```diff
@@ -583,6 +589,21 @@ Vec256<int16_t> inline clamp_min(const Vec256<int16_t>& a, const Vec256<int16_t>
   return _mm256_max_epi16(min_val, a);
 }
 
+template<typename T>
+Vec256<int32_t> inline convert_to_int32(const void* ptr) {
```

Comment:
Please type the pointers here and below instead of using `const void*` and doing `reinterpret_cast` immediately on it